Public/Src/IDE/Debugger/RendererAST.cs

```diff
@@ -4,10 +4,12 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics.ContractsLight;
+using System.Linq;

```

Comment:
Just making some things lazy here...

Reply:
Just so I'm on the same track, the Lazyness enabled here was what was demoed yesterday where not everything loaded if there were thousands of pips or files that were in the logs right?

Reply:
Yes, it (among other things) contributes to that effect.